database/pgx/pgx.go

```diff
@@ -433,10 +447,27 @@ func (p *Postgres) ensureVersionTable() (err error) {
 	// users to also check the current version of the schema. Previously, even if `MigrationsTable` existed, the
 	// `CREATE TABLE IF NOT EXISTS...` query would fail because the user does not have the CREATE permission.
 	// Taken from https://github.com/mattes/migrate/blob/master/database/postgres/postgres.go#L258
-	var count int
-	query := `SELECT COUNT(1) FROM information_schema.tables WHERE table_name = $1 AND table_schema = (SELECT current_schema()) LIMIT 1`
-	row := p.conn.QueryRowContext(context.Background(), query, p.config.MigrationsTable)
+	var row *sql.Row
+	tableName := []byte(p.config.MigrationsTable)
```

Comment:
Why use `[]byte` instead of `string`? Similarly, why not use `re.FindAllStringSubmatch()` below?